website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -341,6 +385,71 @@ Specifies key-value pairs to add to the KV store when the configuration entry is
   - keys: String
   - values: String, integer, or float
 
+### `JWT`
+
+Specifies a JSON Web Token provider configured in a [JWT provider configuration entry](/consul/docs/connect/config-entries/jwt-provider), as well as additional configurations for verifying a service's JWT before authorizing communication between services
+
+#### Values
+
+- Default: None
+- Data type: Map that contains [`JWT{}.Providers`](#jwt-providers)
+
+### `JWT{}.Providers`
+
+Specifies the names of one or more previously configured [JWT provider configuration entries](/consul/docs/connect/config-entries/jwt-provider), which include the information necessary to calidate a JSON web token.
+
+#### Values
+
+- Default: None
+- Data type: List of maps
+
+### `JWT{}.Providers[].Provider`
+
+Specifies the name of a JWT provider defined in the `Name` field of the [`jwt-provider` configuration entry](/consul/docs/connect/config-entries/jwt-provider). You must write the JWT Provider to Consul before referencing it in a service intention.
+
+#### Values
+
+- Default: None
+- Data type: String
+
+### `JWT{}.Providers[].VerifyClaims`
+
+Specifies additional token information to verify beyond what is configured in the JWT provider configuration entry. This map takes the form of a JSON web token claim and a value to match for verification.
+
+#### Values
+
+- Default: None
+- Data type: List of maps that can contain the following parameters:
+- 
+  - [`Path`](#jwt-providers-verifyclaims-path)
+  - [`Value`](#jwt-providers-verifyclaims-value)
+
+### `JWT{}.Providers[].VerifyClaims[].Path`
+
+Specifies the path to the claim in the JSON web token. For more information about JWT claims, refer to the [IETF standards documentation](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1).
+
+#### Values
+
+- Default: None
+- Data type: List of strings that can contain one of the following values:
+
+  - `iss`: Issuer claim
+  - `sub`: Subject claim
+  - `aud`: Audience claim
+  - `exp`: Expiration Time claim
+  - `nbf`: Not Before claim
+  - `iat`: Issued At claim
+  - `jti`: JWT ID claim
```

Comment:
The path can actually be any custom path. It's not limited to just these and it's allowed to have entirely custom claims in the JWT. Is that right @roncodingenthusiast?

(Also, to validate the `iss` or `aud` claims, users should use the `Issuer` and `Audience` fields instead of `VerifyClaims`)

Reply:
this is correct, We don't limit the values of what could go in claims